azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -510,6 +511,13 @@ private void addThreadPoolGauges(final QueuedThreadPool threadPool) {
   private void startWebMetrics() throws Exception {
     this.metricsManager
         .addGauge("WEB-NumQueuedFlows", this.executorManagerAdapter::getQueuedFlowSize);
+
+    // Metric for flows that have been submitted, but haven't started for more than 20 minutes.
```

Comment:
Based on the query it's more accurate to say "Metric for executions that have been submitted, but haven't been dispatched..."

Reply:
If we say "started" those 20 minutes include dispatch and preparation steps not just dispatch which I believe is the intention here.